src/main/java/codeu/controller/Stopwords.java

```diff
@@ -0,0 +1,871 @@
+
+  public static ArrayList<String> getList() {
+    //alphabetical arraylist of hobbies occupations and foods to sort for keywords
+      ArrayList<String> list = new ArrayList<String>();
```

Comment:
Use a set rather than a list would improve the efficiency. Set<String> list = new HashSet<>();